### DISCUSSION TITLE
useFetch call api only once

### DISCUSSION THREAD

**Discussion by danillo-hilbs** [nuxt/nuxt#17285] [Questions] [answered]
as in the image below, I have 3 API calls in a row, and in the network tab you can see that it's calling only once

in the useGetDimensions() im using nuxt method "useFetch"

![image](https://user-images.githubusercontent.com/34604307/179075971-b015a9bb-8fae-428a-970b-a8ab4998e175.png)

**Comment by rudimadima** [comment, marked as answer]
Try to use useAsyncData() with `initialCache: false` option.

**Reply by danielroe** [reply]
`useFetch` also supports setting `initialCache: false`

**Reply by danillo-hilbs** [reply]
> Try to use useAsyncData() with `initialCache: false` option.

thanks, its working with useFetch, use the same config
